### PR TITLE
Add action under price

### PR DESCRIPTION
**Pull request by pgarcess**

### CODE REVIEW

File: WC_Pagantis.php [hunk 1]
```diff
@@ -314,10 +315,38 @@ public function pagantisActivation()
         update_option('woocommerce_pagantis_settings', $settings);
     }
 
+    /**
+     * Product simulator PP o SELECTABLE
+     */
+    public function pagantisAddProductSimulatorPP()
```
Comment (tmeliotpg) [inline WC_Pagantis.php]: This method name is not very easy to read or understand, in my opinion a better name would be addProductSimulatorBeforeCartForm()

File: WC_Pagantis.php [hunk 2]
```diff
@@ -314,10 +315,38 @@ public function pagantisActivation()
...
+        }
+
+        return false;
+    }
+
     /**
      * Product simulator
      */
-    public function pagantisAddProductSimulator()
+    private function pagantisAddProductSimulator()
```
Comment (tmeliotpg) [inline WC_Pagantis.php]: This method name is not very easy to read or understand, in my opinion a better name would be addProductSimulatoAfterCartForm()

**Review by tmeliotpg — CHANGES_REQUESTED** [review]

**Review by tmeliotpg — APPROVED** [review]

File: WC_Pagantis.php (hunk 1 above)
Comment (pgarcess) [reply WC_Pagantis.php]: Done

File: WC_Pagantis.php (hunk 2 above)
Comment (pgarcess) [reply WC_Pagantis.php]: This in the main method, it's called from pagantisAddProductSimulatorAfterCartButton or pagantisAddProductSimulatorAfterPrice, according the action

**Pull request merged**